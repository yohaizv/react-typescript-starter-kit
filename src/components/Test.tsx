import styled from "@emotion/styled/macro";

const Test = styled.div`
  background-color: red;
`;

export default Test;
