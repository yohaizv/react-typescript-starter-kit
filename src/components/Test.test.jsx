import React from "react";
import Test from "./Test";
import TestRenderer from 'react-test-renderer';

describe("Test: ", () => {
  it("render correctly", () => {
    const component = TestRenderer.create(<Test>123</Test>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
