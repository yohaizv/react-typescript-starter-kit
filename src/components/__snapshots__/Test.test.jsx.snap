// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test:  render correctly 1`] = `
.emotion-0 {
  background-color: red;
}

<div
  className="emotion-0 emotion-1"
>
  123
</div>
`;
